packages/react-reconciler/src/ReactChildFiber.new.js

```diff
@@ -280,13 +280,22 @@ function ChildReconciler(shouldTrackSideEffects) {
     // deletions, so we can just append the deletion to the list. The remaining
     // effects aren't added until the complete phase. Once we implement
     // resuming, this may not be true.
+    // TODO (effects) Get rid of effects list update here.
     const last = returnFiber.lastEffect;
     if (last !== null) {
       last.nextEffect = childToDelete;
       returnFiber.lastEffect = childToDelete;
     } else {
       returnFiber.firstEffect = returnFiber.lastEffect = childToDelete;
     }
+    const deletions = returnFiber.deletions;
+    if (deletions === null) {
+      returnFiber.deletions = [childToDelete];
+    } else {
+      deletions.push(childToDelete);
+    }
+    // TODO (effects) Rename this to better reflect its new usage (e.g. ChildDeletions)
+    returnFiber.effectTag |= Deletion;
```

Comment:
You shouldn't need to set this on `childToDelete` anymore, right?

Reply:
Which is nice, since `childToDelete` is a current fiber, so that wouldn't work in a world where the commit phase is concurrent with subsequent renders

Reply:
Oh, I guess you might still need it for passive effects phase, which still uses the effect list. Can remove it after that, though.

Reply:
Oops yeah.